src/Goteo/Model/LegalDocumentType/NIE.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Goteo\Model\LegalDocumenType;
+
+class NIE {
```

Comment:
All these classes (under the LegalDocumentType folder), shouldn't extend from `LegalDocumentType`?

Otherwise they aren't really a `LegalDocumentType`. Plus, they won't have access to the `self::NIE` constant either.